{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -23,7 +23,6 @@
 from drf_yasg import openapi
 
 urlpatterns = [
-    path("", include("home.urls")),
```

Comment:
Isn't this required? What will the user see when the go to the root path of the site with this removed?

Reply:
`Hide existing Django pages from backend scaffold. Make the Django scaffold API-only.` that is to hide pages and make it like an API-only by default